google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -55,6 +55,13 @@
  *   <li>The default service address (bigtable.googleapis.com) and default port (443) are used.
  *   <li>Credentials are acquired automatically through Application Default Credentials.
  *   <li>Retries are configured for idempotent methods but not for non-idempotent methods.
+ *   <li>This settings are mixed for streaming and non-streaming operation. In case of streaming
+ *       operations, RPC timeout applies to each row and for the non-streaming operations, RPC
+ *       timeout considered as deadline for every call.
+ *   <li>Each operation setting accepts RPC and total time out. Here, RPC timeouts are for each
+ *       attempt to get the result whereas total timeout applies to the operation timeout.
+ *   <li>RetryDelayMultiplier controls the change in retry delay. After initial attempt, subsequent
+ *       attempts are calculated by multiplying the retry delay of the previous call.
```

Comment:
This feels out of place. The first  3 bullet points describe the values of the settings, while new 3 points are describing concepts. Also, what are you referring to by `this settings`? Also, it's weird that the explanation for RetryDelayMultiplier is 3 bullet points away from the Retries bullet point.

I think it would be better to link users to read the docs for RetrySettings for a description of the concepts and focus on the values in these docs. (see below)

Reply:
I think with your new suggested explanation, User will get the idea about these Retries and RPC timeouts so I have removed these explanations from the Class level Javadoc. 
Please let me know if we want to add any details here.